README.md

```diff
@@ -80,6 +81,19 @@ All released versions can be found [here](https://github.com/sonatype-nexus-comm
 There are a range of options for installing the R plugin. You'll need to build it first, and
 then install the plugin with the options shown below:
 
+### Easiest Install
+
+Thanks to some upstream work in Nexus Repository (versions newer than 3.15), it's become a LOT easier to install a plugin. To install the `conda` plugin, follow these steps:
```

Comment:
![download](https://user-images.githubusercontent.com/5544326/65071800-fea47780-d93b-11e9-8013-68da8781bd18.jpeg)

COPY PASTA: Remove reference to `conda` and replace with `r`

Reply:
Embarrassing first pull. Updated to remove reference to `conda`.

Reply:
Happens to the best of us. You have now been shamed by the person who gets shamed the most! No worries :)